elements/rh-subnav/rh-subnav.ts

```diff
@@ -97,28 +108,30 @@ export class RhSubnav extends LitElement {
   override render() {
     return html`
       <!-- container, \`<div>\` element -->
-      <nav part="container" aria-label="${this.accessibleLabel}">
+      <nav part="container" 
+           aria-label="${this.accessibleLabel}">
         ${!this.#overflow.showScrollButtons ? '' : html`
           <button id="previous"
                   tabindex="-1"
                   data-direction="start"
-                  aria-label="${this.getAttribute('label-scroll-left') ?? 'Scroll back'}"
+                  aria-label="${this.labelScrollLeft}"
                   ?disabled="${!this.#overflow.overflowLeft}"
```

Comment:
```suggestion
                  ?aria-disabled="${!this.#overflow.overflowLeft}"
```

Instead of `disabled`, could we use `aria-disabled` here so that this element remains focusable, but still disabled?

Same goes for the `disabled` attribute on line ~135.

Reply:
We are going to leave this alone because users can tab through links.